### ISSUE TITLE
Wox error web searches new tab

### ISSUE THREAD

**Issue by jorgeroma** [Wox-launcher/Wox#2857]
[2020-04-23.txt](https://github.com/Wox-launcher/Wox/files/4524487/2020-04-23.txt)

Wox version: 1.3.792
OS Version: Microsoft Windows NT 10.0.18363.0
IntPtr Length: 8
x64: True

Python Path: F:\Anaconda\pythonw.exe
Everything SDK Path: C:\Users\jorge\AppData\Local\Wox\app-1.3.792\Plugins\Wox.Plugin.Everything\EverythingSDK\x64\Everything.dll
Date: 04/23/2020 20:25:18
Exception:
System.ComponentModel.Win32Exception (0x80004005): El sistema no puede encontrar el archivo especificado
   en System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   en System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   en Wox.Plugin.SharedCommands.SearchWeb.NewTabInBrowser(String url, String browserPath) en C:\projects\wox\Wox.Plugin\SharedCommands\SearchWeb.cs:línea 54
   en Wox.Plugin.WebSearch.Main.<>c__DisplayClass9_1.<Query>b__2(ActionContext c) en C:\projects\wox\Plugins\Wox.Plugin.WebSearch\Main.cs:línea 87
   en Wox.ViewModel.MainViewModel.<InitializeKeyCommands>b__16_7(Object index) en C:\projects\wox\Wox\ViewModel\MainViewModel.cs:línea 154
   en System.Windows.Input.CommandManager.TranslateInput(IInputElement targetElement, InputEventArgs inputEventArgs)
   en System.Windows.UIElement.OnKeyDownThunk(Object sender, KeyEventArgs e)
   en System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   en System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   en System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   en System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   en System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   en System.Windows.Input.InputManager.ProcessStagingArea()
   en System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   en System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   en System.Windows.Interop.HwndKeyboardInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawKeyboardActions actions, Int32 scanCode, Boolean isExtendedKey, Boolean isSystemKey, Int32 virtualKey)
   en System.Windows.Interop.HwndKeyboardInputProvider.ProcessKeyAction(MSG& msg, Boolean& handled)
   en System.Windows.Interop.HwndSource.CriticalTranslateAccelerator(MSG& msg, ModifierKeys modifiers)
   en System.Windows.Interop.HwndSource.OnPreprocessMessage(Object param)
   en System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   en System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by bao-qian** [comment]
please describe your issue in plain English and give a list of steps to reproduce the issue.
otherwise we won't understand you problem.
